train.py

```diff
@@ -208,20 +230,18 @@ def k_fold_train_process(config, model, k_fold, paths, dataset, cp_callback, ini
         if not config['k_fold_merge_model']:
 
             # train all k-fold on one model
-            model, history_curr = train_process(config, model, paths_train_img, paths_train_label, paths_val_img,
```

Comment:
redefinition can't be accepted. history needs to be saved as list over k-fold runs